src/components/organisms/Vault/vault.organism.tsx

```diff
@@ -263,11 +263,61 @@ const Vault: React.FC<IVaultProps> = (props) => {
 
   const buyTokenUrl = buyTokenUrlByTokenAndNetwork[token][chainId];
 
+  const setAvailableValueHandler = (options: ISetBalanceOptions) => {
```

Comment:
Name kinda confuses me, sounds like there's some "availableValue" variable and this is setting it. Maybe something like "updateBalanceInput" is more appropriate since it's agnostic to what it really is?